runner/runner.go

```diff
@@ -2077,6 +2090,7 @@ retry:
 			"PageType": r.errorPageClassifier.Classify(respData),
 			"pHash":    pHash,
 		},
+		TechnologyDetails: technologyDetails,
```

Comment:
How about moving this under "KnowledgeBase" as "Technologies"?

Reply:
Actually, we only want this to be available from sdk. so we can't add it there